.devcontainer/Dockerfile

```diff
@@ -29,6 +29,7 @@ COPY activerecord/activerecord.gemspec /tmp/rails/activerecord/
 COPY activestorage/activestorage.gemspec /tmp/rails/activestorage/
 COPY activesupport/activesupport.gemspec /tmp/rails/activesupport/
 COPY railties/railties.gemspec /tmp/rails/railties/
+COPY tools/releaser /tmp/rails/tools/releaser
```

Comment:
Can we copy only the gem spec? That is all we need

Reply:
Done https://github.com/rails/rails/blob/7f13428e3baf23c94917c9105a15e0c94e2be6f1/.devcontainer/Dockerfile#L32